xdem/spstats.py

```diff
@@ -91,6 +90,73 @@ def random_subset(dh: np.ndarray, coords: np.ndarray, nsamp: int):
 
     return dh_sub, coords_sub
 
+def create_circular_mask(h: float, w: float, center: Optional[list[float]] = None, radius: Optional[float] = None) -> np.ndarray:
+    """
+    Create circular mask on a raster
+
+    :param h: height of array
```

Comment:
Maybe use a shape directly? It's simpler as one can use `array.shape` as input instead.

Reply:
If this is done, the type would be `tuple[int, ...]` as an array's number of dimensions is never known by the type hinting. I just mention it here because I've been stuck on it myself!

Reply:
Done